packages/volto/Makefile

```diff
@@ -66,6 +66,10 @@ install: ## Set up development environment
 	pnpm i
 	make build-deps
 
+.PHONY: i18n
+i18n: ## Extract and compile translations
+	pnpm i18n
```

Comment:
This belongs below the `.PHONY: build-deps` target, under the `## Build` section.